### ISSUE TITLE
Next Steps for WC Payments in Core

### ISSUE THREAD

**Issue by adrianduffell** [Automattic/wc-calypso-bridge#744]
In #721, the Payments menu item was added by default to business/ecomm plan sites.

The next steps are:

- [ ] Expand it to additional countries (TBA approximately 6-7).
- [ ] Content changes (TBA).
- [ ] Minor design changes. (TBA).

**Comment by adrianduffell** [comment]
@pmcpinto @elizaan36 Could you please add more details about these items once available? Ghidorah can work on this in Cycle 17.

**Comment by elizaan36** [comment]
👋 Here's an updated layout for the welcome screen.

The actions have been moved to the right side of the header card, and the incentive is separated into its own card between the header and FAQ. Everything else is consistent with i1.
![WCPay welcome](https://user-images.githubusercontent.com/5121465/132565535-17a4c0d6-42d4-4cc8-9106-e4026a9a8a79.png)

And the updated content for the FAQ is in [this doc](https://docs.google.com/document/d/1txfcYxA_1ha8QhOTRdXtZ7QLhq5enIAvxPlNGI1v_X8/edit#).

**Comment by pmcpinto** [comment]
I updated the issue description with the acceptance criteria. @elizaan36 do you think we should move forward with the "remind me later" idea?

**Comment by elizaan36** [comment]
Absolutely! So if the user navigates away from the welcome screen without taking action (either removing Payments or Installing), an Inbox note is sent as a reminder for follow up a few days later?

**Comment by elizaan36** [comment]
aha. In that case I don't think a "Remind me later" action would be very useful for merchants. If users want to come back to it later, they can navigate away until they're ready, that's why I thought you were referring to an automatic reminder. I don't see a benefit in adding another choice for merchants on this screen.

The only place it would make sense is in the "Remove" modal, but that would add an additional action to a modal that already requires a lot of cognitive load.
<img width="657" alt="Screen Shot 2021-09-10 at 12 19 25 PM" src="https://user-images.githubusercontent.com/5121465/132892763-35410895-582a-4fc8-8315-9b2f45634744.png">

**Comment by pmcpinto** [comment]
>  I don't see a benefit in adding another choice for merchants on this screen.
@elizaan36 
I wouldn't survive one day without a "remind later" feature in some products like Gmail 😅

I thought a bit more about this and it makes sense to keep it simple with an automatic reminder. Can you work on the inbox copy? what do you think about triggering the reminder 3 days after they view the welcome screen?

**Comment by elizaan36** [comment]
Yep, will think about copy and have a look at the notes we already have for WCPay. I think 3 days sounds reasonable.

**Comment by ilyasfoo** [comment]
Hey @elizaan36, I've looked at the [design](https://user-images.githubusercontent.com/5121465/132892372-ecd703d8-3ae0-4ecc-b2f0-a23925b69825.png) and it seems the banner copy is using another promo's copy, which I think was unintended. I've made the changes but I retain existing promo copy:

![image](https://user-images.githubusercontent.com/3747241/134888608-8534e262-4c9c-470f-81b2-b8aac4d4f1da.png)

Can you check if there's anything off for the copy in the banner and the main CTA card? Thanks!

**Comment by elizaan36** [comment]
Looks good, my only question would be whether the "Save $800 in fees..." is still accurate. (From the body copy in the first card)

Also, the FAQ card seems to be using slightly different text weight than the others. Want to make sure we're not introducing any new text styles there, just use the same as what's available/to match other text components in the other cards.

**Comment by ilyasfoo** [comment]
@elizaan36 Good catch, I'll change it to "Save up to $125,000 in fees" to reflect this promo. 

>Also, the FAQ card seems to be using slightly different text weight than the others. Want to make sure we're not introducing any new text styles there, just use the same as what's available/to match other text components in the other cards.

Indeed, I've added a little bit of weight for the accordion title and FAQ card header. I'll change it back to default.

**Comment by ilyasfoo** [comment]
@elizaan36 Ah, ok!

**Comment by ilyasfoo** [comment]
Updated, let me know if anything else needs to be changed!

![image](https://user-images.githubusercontent.com/3747241/135011755-c3a369e4-fa9e-4215-b68e-37c33573e6d5.png)

**Comment by elizaan36** [comment]
Looks perfect to me, just one really small nitpick to remove the period after "Terms of service" in the header.

**Comment by ilyasfoo** [comment]
Sure thing!

**Comment by pmcpinto** [comment]
@adrianduffell what's the note-name of the inbox reminder? There's one inbox called `wc-calypso-bridge-payments-remind-me-later` but it's getting 0 views, so I'm not sure if this is the right message

**Comment by adrianduffell** [comment]
@pmcpinto Hmm `wc-calypso-bridge-payments-remind-me-later` is the right one.

I'll test this out further and see if the note appears for me.

**Comment by pmcpinto** [comment]
@adrianduffell thanks! Let me know when you have news about the inbox message.

**Comment by moon0326** [comment]
@adrianduffell @ilyasfoo @pmcpinto 

I tested the change locally and it worked as expected, but the test on wordpress.com did not work.

`wc-calypso-bridge` did not register `wc_calypso_bridge_daily` cron event, which is required to create the note.

We use [register_activation_hook](https://github.com/Automattic/wc-calypso-bridge/blob/master/includes/class-wc-calypso-bridge-events.php#L27) to register the corn event. My best guess is that the hook does not trigger because `wc-calypso-bridge` is part of MU plugins.

I think we need to find a different way to register for the corn event. I'll look into it.

**Comment by ilyasfoo** [comment]
Oops, I really thought I've been thorough in my WPCOM test, sorry to have missed that. I may also look into it further

**Comment by ilyasfoo** [comment]
Hi @pmcpinto, the fix for inbox note reminder is deployed

**Comment by pmcpinto** [comment]
👍 thanks for the heads up!
